ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -1452,17 +1452,20 @@ IHqlExpression * SequenceNumberAllocator::createTransformed(IHqlExpression * exp
             HqlExprArray args;
             ForEachChild(i, expr)
             {
-                OwnedHqlExpr next = transform(expr->queryChild(i));
-                if (!next->isAction())
+                LinkedHqlExpr cur = expr->queryChild(i);
+                if (!cur->isConstant())
                 {
-                    if (!next->isConstant())
-                        next.setown(createValue(no_evaluate_stmt, makeVoidType(), next.getClear()));
-                    else
-                        next.clear();
-                }
+                    if (!cur->isAction())
+                    {
+                        HqlExprArray args;
```

Comment:
This 'args' variable shadows the 'args' in line 1452.  Potential to cause confusion/problems.  May be rename.